resources/profiles/Anycubic.ini

```diff
@@ -922,19 +939,23 @@ inherits = *0.20mm_mega*;*supported_mega*
 [print:*0.30mm_mega*]
 inherits = *common_mega*
 bottom_solid_layers = 4
+bridge_flow_ratio = 0.95
 external_perimeter_extrusion_width = 0.6
 external_perimeter_speed = 35
 extrusion_width = 0.5
-fill_pattern = grid
```

Comment:
I copied cubic form the MK2 "FAST" profile and would like to use it here because it prints faster than gyroid and also is a three dimensional pattern. Printing time between cubic and grid is quite similar.

Reply:
this profile is called QUALITY so it suggests that it is like the Prusa QUALITY profiles
I have taken some values from the newest MK3S+ profiles

please create an additional FAST profile

Something about infills:
https://blog.prusaprinters.org/everything-you-need-to-know-about-infills_43579/

Reply:
I think in the current state with this patch is that all profiles that are called QUALITY also get `fill_pattern = gyroid`. This change here should only impact the profile that's called DARFT.
But maybe it might make more sense to put this not to the `*0.30mm_mega*` template, but in the `0.30mm DRAFT  @MEGA` section, although in the end there will be no visible difference to the users.

Reply:
ah, it was my fault :facepalm: 
GitHub has hide the lines 913 to 924 so I thought it would impact the ```0.20mm QUALITY @MEGA``` profile.

grid is fine for the draft profile

Reply:
Yeah, I agree the github diff display is not really user-friendly here. I'm closing this as I think everything is resolved now.